src/packs.c

```diff
@@ -104,6 +102,7 @@ int download_subscribed_packs(struct list *subs, bool required)
 		return -ENOSWUPDSERVER;
 	}
 
+	printf("Downloading packs...\n");
```

Comment:
I noticed there is a "Downloading packs" printf in bundle.c. Can that one be removed?